src/ScanItems.js

```diff
@@ -227,12 +256,15 @@ class ScanItems extends React.Component {
       return;
     }
 
-    const item = await this.fetchItem(barcode);
+    const checkoutItems = await this.fetchItems(barcode);
+    const checkoutItem = checkoutItems[0];
 
-    if (!item) {
-      this.checkout(barcode);
+    if (checkoutItems.length > 1) {
+      this.setState({ items: checkoutItems });
+    } else if (isEmpty(checkoutItems)) {
+      this.checkout(checkoutItem.barcode);
```

Comment:
Is it correct that `checkoutItem` will be `undefined` if `checkoutItems` is empty, because 
`const checkoutItem = checkoutItems[0];`?
I think we can get an error here or broke something, when will try to read `barcode` of `undefined`.
Am I right, or I miss something?

Reply:
Thank you @Stanislau-Snapok , good catch.
In case the list is empty we need to pass the originally enterred barcode to produce the correct notification that there is no items for such barcode.
Fixed.

Reply:
To be honestly, it still looks a little bit woried for me.
`this.checkout(checkoutItem?.barcode || barcode)`
In my opinion, according to my previous comment, we will never have a case, when this can work `checkoutItem?.barcode`.
So I suggest leave just `barcode`, as it was before.
What do you think @Godlevskyi?

Reply:
@Stanislau-Snapok  the case is the following
1) the wildcard search is on
2) we have item with barcode 1231
3) We search for barcode 123 and receive an item with barcode 1231
4) We need to checkout item with its barcode 1231 and not with the one we searched for with wildcard search

Reply:
Is `checkoutItems` will be still empty in this case?
If we need to checkout one item, it will be doing according to 267 string, is it right?
Because our fist `if` will not work (we have not more than one item), the second one will not work too, because if we receive atleast one item, our `checkoutItems` can't be empty, and our `checkoutItem` will be equal to `checkoutItems[0]`.
Is it correct?

Reply:
Yeap, you are right, we can leave just barcode here